source/includes/steps-install.yaml

```diff
@@ -0,0 +1,67 @@
+title: "Install Xcode from the App Store."
+ref: install-xcode
+content: |
+  
+  Install Xcode -- it takes awhile. In the meantime, you can follow the rest of the steps.
+
+---
```

Comment:
Do we need XCode, XQuartz, and inkscape?

Reply:
xquartz and inkscape are not required, no.